mixed/flink/flink-common/src/main/java/com/netease/arctic/flink/catalog/factories/ArcticCatalogFactoryOptions.java

```diff
@@ -18,6 +18,7 @@
 
 package com.netease.arctic.flink.catalog.factories;
 
+import static com.netease.arctic.ams.api.properties.CatalogMetaProperties.TABLE_FORMATS;
```

Comment:
This class is for a unified catalog or a mixed-format catalog?

If it is for mixed-format, it is suggested to rename the class to MixedCatalogFactoryOptions and move it to the mixed package.

Reply:
This Options class is for both a unified catalog and a mixed catalog. I prefer to rename `ArcticCatalogFactoryOptions` to `CatalogFactoryOptions`. WDYT?